src/pool/cli.c

```diff
@@ -48,6 +198,10 @@ dc_pool_init(void)
 	uint32_t		ver_array[2] = {DAOS_POOL_VERSION - 1, DAOS_POOL_VERSION};
 	int			rc;
 
+	d_getenv_bool(DAOS_CLIENT_METRICS_ENABLE, &daos_client_metric);
```

Comment:
Oh, that seems strange. I haven't looked closely at Di's base patch. I wonder why he put `dc_tm_init()` at the bottom, rather than calling it before all of the other `_init()` functions. I'll experiment with moving things around to see if we can get rid of those redundancies.

Reply:
This went OK. I was able to get rid of those extra getenvs... I suspect that they were a holdover from earlier in the development process.